Resources/views/menu.html.twig

```diff
@@ -0,0 +1,3 @@
+{% extends 'knp_menu.html.twig' %}
+
+{% block label %}{{ parent()|trans({}, item.getExtra('translation_domain', 'messages')) }}{% endblock %}
```

Comment:
what is the space_locale? is that something custom from your project? i would expect that we do just use the request locale here?